examples/neuralNetwork-sequence-weather-prediction/index.html

```diff
@@ -0,0 +1,33 @@
+<!--
+  👋 Hello! This is an ml5.js example made and shared with ❤️.
+  Learn more about the ml5.js project: https://ml5js.org/
+  ml5.js license and Code of Conduct: https://github.com/ml5js/ml5-next-gen/blob/main/LICENSE.md
+ 
+  This example demonstrates Training a Weather Predictor through ml5.neuralNetwork with sequeceRegression Task.
```

Comment:
@mop9047 I enjoyed the simplicity of the example. For making the visual output more interesting: how about a bar in a bar-graph gets drawn per hour predicted? (Since you know the training data, you wouldn't need to deal with automatically scaling the y-axis, and instead used some fixed min/max.)
You could divide the width of the canvas into 24 equally-spaced columns, and draw one by one, as the user predicts one hour after the next.